lib/LaTeXML/Common/Model/RelaxNG.pm

```diff
@@ -105,7 +105,7 @@ sub filterNames {
       && ((($name =~ /^!(.*)$/) && !(defined $$hash{$1}))    # Negated name, but name not present?
         || (!defined $$hash{ '!' . $name }))) {              # Or negation of name not present
       $filtered{$name} = 1; } }
-  return sort keys %filtered; }
+  return (sort keys %filtered); }
```

Comment:
the new parens around `sort` were motivated by Perl Critic complaining loudly that `sort` should not follow a return (severity 5, [docs](https://metacpan.org/pod/Perl::Critic::Policy::Subroutines::ProhibitReturnSort))